ghost/core/test/utils/fixtures/data-generator.js

```
@@ -1431,13 +1431,17 @@ DataGenerator.forKnex = (function () {
-    ];
+    const tags = DataGenerator.Content.tags.map((tag, index) => {
+        const adjustedTime = new Date(new Date().getTime() + index * 1000);
+
+        return createTag({
+            ...tag,
+            created_at: adjustedTime,
+            updated_at: adjustedTime
+        });
+    });
+
+    console.log('tags', tags);
```

ESLint

[error] Unexpected console statement.

(no-console)